### ISSUE TITLE
[APP SUBMITTED]: SEARCHQUEUE-DAILY-SEARCH :: qbittorrent: Unable to send Torrent: Return code undefined

### ISSUE THREAD

**Issue by wishx** [SiCKRAGETV/sickrage-issues#1870]
### INFO
Python Version: **2.7.6 (default, Mar 22 2014, 22:59:56) [GCC 4.8.2]**
Operating System: **Linux-3.13.0-24-generic-x86_64-with-LinuxMint-17-qiana**
Locale: unknown
Branch: **master**
Commit: SiCKRAGETV/SickRage@52a6c31cdf545c4e2fd22925a92a3d0824c429cf
Link to Log: https://gist.github.com/cf53395e0f3dcb977a73
### ERROR
```
SEARCHQUEUE-DAILY-SEARCH :: qbittorrent: Unable to send Torrent: Return code undefined
```
---
_STAFF NOTIFIED_: @SiCKRAGETV/owners @SiCKRAGETV/moderators

**Comment by neoatomic** [comment]
@wishx Problems seems to be the new qbittorrent 3.2+ If you downgrade to version 3.1 it should work. For more temp fixes/information you can look here. https://github.com/SiCKRAGETV/sickrage-issues/issues/1723

**Comment by drinkyouroj** [comment]
### INFO
Python Version: **2.7.10 (default, Sep  7 2015, 13:51:49) [GCC 5.2.0]**
Operating System: **Linux-4.1.9-2-lts-x86_64-with-glibc2.2.5**
Locale: UTF-8
Branch: **master**
Commit: SiCKRAGETV/SickRage@d1118cb1c9114778535b85b7e59758fde57ad9a5
Link to Log: https://gist.github.com/49860ef0b806de0b25bd
### ERROR
```
SEARCHQUEUE-BACKLOG-255569 :: [BTN] :: qbittorrent: Unable to send Torrent: Return code undefined
```
---
_STAFF NOTIFIED_: @SiCKRAGETV/owners @SiCKRAGETV/moderators

**Comment by beberito** [comment]
### INFO
Python Version: **2.7.10 (default, Aug 19 2015, 09:18:54) [GCC 4.6.3]**
Operating System: **Linux-3.12.20-x86_64-with-glibc2.2.5**
Locale: UTF-8
Branch: **master**
Commit: SiCKRAGETV/SickRage@a3a3b92f0fb813beffa5473576396b0e49aa04ba
Link to Log: https://gist.github.com/4dfb201545508371e79b
### ERROR
```
SEARCHQUEUE-MANUAL-250487 :: [Rarbg] :: qbittorrent: Unable to send Torrent: Return code undefined
```
---
_STAFF NOTIFIED_: @SiCKRAGETV/owners @SiCKRAGETV/moderators

**Comment by fernandog** [comment]
@beberito please check if the torrent is already in qbitorrent.

**Comment by beberito** [comment]
@fernandog : no torrent in qbittorrent
It's on a ASUSUTOR nas with qbittorrent 3.2.3

**Comment by fernandog** [comment]
do you have `American.*.Story.S05E02.PROPER.720p.HDTV.x264-KILLERS[rartv]` in your download folder?

**Comment by beberito** [comment]
no, nothing

**Comment by fernandog** [comment]
@beberito try to add this to qbittorrent:


`magnet:?xt=urn:btih:e1b13d8e055b7388b4ee6ad77bf081c5094ce169&dn=American.Horror.Story.S05E02.720p.HDTV.x264-FLEET%5Brartv%5D&tr=http%3A%2F%2Ftracker.trackerfix.com%3A80%2Fannounce&tr=udp%3A%2F%2F9.rarbg.me%3A2710&tr=udp%3A%2F%2F9.rarbg.to%3A2710&tr=udp%3A%2F%2Fopen.demonii.com%3A1337%2Fannounce`

**Comment by beberito** [comment]
it's working, maybe a problem with the connection of qbittorrent in sickrage ?

**Comment by fernandog** [comment]
not sure

**Comment by beberito** [comment]
I think i'll try solution in https://github.com/SiCKRAGETV/sickrage-issues/issues/1625 : connection 127.0.0.1 without authentification

**Comment by fernandog** [comment]
let me know if works

**Comment by beberito** [comment]
I already set localhost in sickrage config but i disabled authentification for localhost in qbittorrent and remove login/passowrd in sickrage on it's work now (with manual download via sickrage)
Thank you.
But i'll wait tomorrow to check if automatic download works :-)

**Comment by fernandog** [comment]
Is there any kind of block setting when not accessing from localhost?

**Comment by beberito** [comment]
i don't think so, ports are open via ezrouter in asustor config and sickrage and qbittorrent are accessible from outside.

**Comment by Quiksmage** [comment]
### INFO
Python Version: **2.7.9 (default, Dec 10 2014, 12:28:03) [MSC v.1500 64 bit (AMD64)]**
Operating System: **Windows-2012Server-6.2.9200**
Branch: **master**
Commit: SiCKRAGETV/SickRage@136f531f93093a5287b588ca37e19c11153f47a4
No Log available with ERRORS: 
### ERROR
```
SEARCHQUEUE-DAILY-SEARCH :: qbittorrent: Unable to send Torrent: Return code undefined
```
---
_STAFF NOTIFIED_: @SiCKRAGETV/owners @SiCKRAGETV/moderators

**Comment by Quiksmage** [comment]
### INFO
Python Version: **2.7.9 (default, Dec 10 2014, 12:28:03) [MSC v.1500 64 bit (AMD64)]**
Operating System: **Windows-2012Server-6.2.9200**
Branch: **master**
Commit: SiCKRAGETV/SickRage@136f531f93093a5287b588ca37e19c11153f47a4
Link to Log: https://gist.github.com/332c782325f86365752f
### ERROR
```
SEARCHQUEUE-MANUAL-134161 :: [KickAssTorrents] :: qbittorrent: Unable to send Torrent: Return code undefined
```
---
_STAFF NOTIFIED_: @SiCKRAGETV/owners @SiCKRAGETV/moderators

**Comment by Quiksmage** [comment]
### INFO
Python Version: **2.7.9 (default, Dec 10 2014, 12:28:03) [MSC v.1500 64 bit (AMD64)]**
Operating System: **Windows-2012Server-6.2.9200**
Branch: **master**
Commit: SiCKRAGETV/SickRage@136f531f93093a5287b588ca37e19c11153f47a4
Link to Log: https://gist.github.com/fd49de99f6b6ac05396d
### ERROR
```
SEARCHQUEUE-MANUAL-134161 :: [KickAssTorrents] :: qbittorrent: Unable to send Torrent: Return code undefined
```
---
_STAFF NOTIFIED_: @SiCKRAGETV/owners @SiCKRAGETV/moderators

**Comment by Quiksmage** [comment]
### INFO
Python Version: **2.7.9 (default, Dec 10 2014, 12:28:03) [MSC v.1500 64 bit (AMD64)]**
Operating System: **Windows-2012Server-6.2.9200**
Branch: **master**
Commit: SiCKRAGETV/SickRage@136f531f93093a5287b588ca37e19c11153f47a4
Link to Log: https://gist.github.com/981da02af6722cd54c6f
### ERROR
```
SEARCHQUEUE-MANUAL-134161 :: [KickAssTorrents] :: qbittorrent: Unable to send Torrent: Return code undefined
```
---
_STAFF NOTIFIED_: @SiCKRAGETV/owners @SiCKRAGETV/moderators

**Comment by Quiksmage** [comment]
### INFO
Python Version: **2.7.9 (default, Dec 10 2014, 12:28:03) [MSC v.1500 64 bit (AMD64)]**
Operating System: **Windows-2012Server-6.2.9200**
Branch: **master**
Commit: SiCKRAGETV/SickRage@136f531f93093a5287b588ca37e19c11153f47a4
Link to Log: https://gist.github.com/88fe579f4c8a684f624d
### ERROR
```
SEARCHQUEUE-MANUAL-254243 :: [KickAssTorrents] :: qbittorrent: Unable to send Torrent: Return code undefined
```
---
_STAFF NOTIFIED_: @SiCKRAGETV/owners @SiCKRAGETV/moderators

**Comment by fernandog** [comment]
@JustMaier do you still have some free time to SR? If so, can you update code to support new qbittorrent version? thanks!

**Comment by fernandog** [comment]
@beberito 

can you check your history to see if there's multiple snatches from the same episodes? 
Do you have notification enabled for snatch?

**Comment by beberito** [comment]
@fernandog For me, problem is solved with "authentification disabled for localhost" in qbittorrent conf and removing login/password in sickrage conf.

**Comment by fernandog** [comment]
@beberito 
ok. but do you have notification enabled for snatch?
and please can you check your history to see if there's multiple snatches from the same episodes?
Just change history limit to "all" see if anything pops.

**Comment by Quiksmage** [comment]
The easiest way to trigger the error on the already have torrent in client return code, is to manually search for an episode you already have snatched (which will usually pick the same exact torrent if you click yes when searching). I don't think I've personally ever had it keep on snatching as even though you get the error, it will mark it as snatched in sickrage. Usually when you add a torrent that already exists, it will merge the trackers from the two torrents and I cannot tell if this is the case with sickrage to qbittorrent currently.

**Comment by beberito** [comment]
@fernandog : No multiples snatches for me
